.travis.yml

```diff
@@ -24,7 +24,6 @@ install:
   - npm --version
   - npm install -g elm@$ELM_VERSION
   - npm install -g elm-test
-  - npm install -g elm-doc-test
```

Comment:
why was this removed? 🤔

Reply:
It wasn't working for me, likely due to an elm-doc-test bug when there aren't any doc tests. It's a good template, but wasn't used by this package. I'd prefer a solution that made it work, but I was too lazy at the time. Will work on a proper solution if you want.

Reply:
It was used by this package. There are multiple doc tests. They likely need to be upgraded to use elm-verify-examples: but they should be upgraded, not dropped.

Reply:
I see that elm-doc-test has been renamed to elm-verify-examples.

elm-doc-test 4.0.0 creates elm files that don't compile, when there are no doc tests.

The latest version of elm-verify-examples does not have that problem. I'll push a change to use that instead.

Reply:
thanks!

Reply:
The change for elm-verify-examples was easy, but I'm seeing the same problem with the Linux Travis build that stopped this PR, it runs the (empty. really) elm-verify-examples tests, and then hangs. I'm going to debug that in my fork before making a new PR.

Reply:
Well, elm-verify-examples doesn't generate code that won't compile when a file contains no doc tests, but it does create code that signal a test error. So, since there really are NOT any doc tests, it was creating four errors in elm-test. I effectively commented those out of tests/elm-verify-examples.json.

The reason for the timeout was a big example S3 return XML with 1,000 files. That takes over 4 seconds on my brand new 4-core iMac. I shortened it to 100, and it no longer times out.

My changes are awaiting approval from myrho, before you'll see them here.

Reply:
Thee are doc tests: https://github.com/eeue56/elm-xml/blob/master/src/Xml/Decode.elm#L171

Reply:
Ah. Perhaps at one time ">>>" was the magic prefix for a doc test. Now it is "-->". I've never used them, so I have no memory of when that changed.

https://github.com/stoeffel/elm-verify-examples

Reply:
Yep. So in order to switch, you'll need to change the syntax of the examples

Reply:
I figured out what happened. Referencing https://github.com/stoeffel/elm-verify-examples/commits/master :

Jun 14, "Proposal new dsl" changed the syntax.
Jun 16, "Version 4.0.0" (of elm-doc-test) released.
Jun 16, "Renaming to elm-verify-examples".
Jun 17, "Version 1.0.1" (of elm-verify-examples) released.

The Travis script was getting elm-doc-test version 4.0.0, with the new syntax, hence failing.

I pushed a change to @myrho's fork to update all the comment tests to the new syntax. Travis is happy in Linux, but still waiting to run the macOS test.

Hopefully, @myrho will check in soon, fix the other issues, and commit to add his and my fixes to this PR.

I also added syconfcpus to the Travis script, to speed "elm make" from minutes to seconds. Yes. It's that amazing.

Reply:
Just open a new PR against here, no need to merge into the fork